.changeset/popover.md

```diff
@@ -0,0 +1,69 @@
+---
+'@leafygreen-ui/popover': major
+---
+
+[LG-4445](https://jira.mongodb.org/browse/LG-4445): Replaces `usePortal` prop with `renderMode` prop with values of `'inline'`, `'portal'`, and `'top-layer'`. `renderMode="inline"` and `renderMode="portal"` are deprecated, and all popover elements should migrate to using the top layer. The old default was `usePortal = true`, and the new default is `renderMode = 'top-layer'`.
+  - When `renderMode="top-layer"` or `renderMode` is `undefined`, the popover element will render in the top layer using the [popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API)
```

Comment:
With the top layer, the popover is still a child of the trigger. Is that correct? If so, do we mention that anywhere?

Reply:
I think It's at least worth mentioning that `usePortal` added the popover to the end of the DOM, but the top layer does not.

Reply:
Maybe something like:

> When renderMode="top-layer" or renderMode is undefined, the popover element will render in the top layer using the [popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API). Unlike the usePortal=true approach, which portals the popover element into a new `<div>` appended to the body, the top layer allows the popover to render in a special layer above all other elements without the need for an additional DOM node.

Reply:
I think we should spell out the difference to make it easier for consumers to digest.